README.md

```diff
@@ -81,18 +101,14 @@ environment variables.  See the
         file: quote.txt
```

Comment:
If you don't require any Docker secrets, then this section can be deleted.

Reply:
Resolved in c15795b18e4337758b108e73d9213e84d30e4ba6